UI/GameSettingsScreen.cpp

```diff
@@ -912,6 +916,8 @@ UI::EventReturn GameSettingsScreen::OnChangeNickname(UI::EventParams &e) {
 	if (System_InputBoxGetString("Enter a new PSP nickname", g_Config.sNickName.c_str(), name, name_len)) {
 		g_Config.sNickName = name;
 	}
+#elif defined(ANDROID)
+	System_SendMessage("inputbox", ("nickname :" + g_Config.sNickName).c_str());
```

Comment:
Do these need to be translated?  I think the title is shown to the user.

Also, why the trailing space before `:`?

-[Unknown]